uefi/src/main.rs

```diff
@@ -32,8 +48,35 @@ use x86_64::{
     PhysAddr, VirtAddr,
 };
 
+pub mod portable_acpi;
+use portable_acpi::PortableAcpiTables;
 mod memory_descriptor;
 
+pub static ACPI_TABLES: OnceCell<PortableAcpiTables> = OnceCell::uninit();
```

Comment:
UEFI is single threaded, so we don't need a spinlock. `VeryUnsafeCell` like `SYSTEM_TABLE` would probably work fine.

Reply:
Also `pub` in an executable doesn't have any effect.